copyright/copyright.js

```diff
@@ -0,0 +1,106 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+
+const glob = require('glob')
+const fs = require('fs')
+
+const copyright = {
+    items: [],
```

Comment:
What are these items? :)  I think they're individual file paths, so renaming them to such would make this variable alot easier to "digest".  Maybe `filePaths: []` or even `filesToProcess: []`?